### ISSUE TITLE
GitLab CI from example doesn ot work out of the box

### ISSUE THREAD

**Issue by JacksonChen666** [cstate/cstate#234]
**Describe the bug**
The docker image pulled in does not contain git, and so hugo fails to build the site.

**Reproduction steps**
1. Clone example cstate
2. Push to gitlab (e.g. `git push git@gitlab.com:<username>/<repo-name>.git` replace contained angle brackets with appropriate values, creating of repo not required beforehand)
3. Experience CI failure

**Expected behavior**
The GitLab CI working out of the box

**Screenshots**
[![Screen Shot 2022-06-13 at 14 44 04](https://user-images.githubusercontent.com/50755746/173358403-75fd1e57-9ab0-4f65-b899-0888f4930cc0.png)](https://gitlab.com/JacksonChen666/status.jacksonchen666.gitlab.io/-/jobs/2582179511)

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
 N/A

**Additional context**
The images are stored [here]()

**Comment by mistermantas** [comment]
Hi. Have you got a proposed solution? I don't think I set up the Docker instructions myself so I'm not exactly sure how to help at the moment.

Best,

Mantas Vilčinskas

**Comment by JacksonChen666** [comment]
@mistermantas yes, i suddenly now have a proposed solution, and that is bumping the version up much more.

i assumed that there was probably a git executable later in the versions (in some github community maintained hugo-extended). git was actually there, and [some changes later](https://gitlab.com/JacksonChen666/status.jacksonchen666.gitlab.io/-/commit/984f3eccc3df6f75615b80eec4f345fefddd3d3c), [ci is working fine](https://gitlab.com/JacksonChen666/status.jacksonchen666.gitlab.io/-/jobs/2582438114). please see cstate/example# and move this issue to cstate/example (this is the wrong repo to report the issue on, oops)
